protocols/CreateInhouseRnaSeqProjects.sh

```diff
@@ -130,6 +130,7 @@ fi
 echo "before run second rocket"
 echo pwd
 
+module load Molgenis-Compute/v16.05.1-Java-1.8.0_45
```

Comment:
Why not loading RNA pipeline instead (now you can get differences between the RNA pipeline compute and loading this hardcoded compute)